README.md

```diff
@@ -38,28 +41,50 @@ Version of deployed RollupAdminLogic: v1.1.1
 Version of deployed RollupUserLogic: v1.1.1
 This deployment can be upgraded to v1.2.1 using NitroContracts1Point2Point1UpgradeAction
 ```
-For other networks, replace `arb1` with the network name and configure INFURA_KEY or the rpc in hardhat.config.ts
 
+For other networks, replace `arb1` with the network name and configure INFURA_KEY or the rpc in hardhat.config.ts
 
 ## Nitro Contracts Upgrades
-*This section is also referenced in the documentation on ["How to upgrade ArbOS on your Orbit chain"](https://docs.arbitrum.io/launch-orbit-chain/how-tos/arbos-upgrade)*
 
-For ArbOS upgrades, a pre-requisite is to deploy new Nitro contracts to the parent chain of your Orbit chain before scheduling the ArbOS upgrade. These contracts include the rollup logic, fraud proof contracts, and interfaces for interacting with Nitro precompiles. 
+_This section is also referenced in the documentation on ["How to upgrade ArbOS on your Orbit chain"](https://docs.arbitrum.io/launch-orbit-chain/how-tos/arbos-upgrade)_
+
+For ArbOS upgrades, a pre-requisite is to deploy new Nitro contracts to the parent chain of your Orbit chain before scheduling the ArbOS upgrade. These contracts include the rollup logic, fraud proof contracts, and interfaces for interacting with Nitro precompiles.
+
+### Nitro contracts 2.1.0 (for ArbOS 31 Bianca)
+
+The [`nitro-contracts 2.1.0` upgrade action](scripts/foundry/contract-upgrades/2.1.0) will deploy `nitro-contracts v2.1.0` contracts to your Orbit's parent chain. Note that this action will only work for chains with `nitro-contracts v1.2.1` or `nitro-contracts v1.3.0`. ArbOS 31 Bianca, shipped via [Nitro v3.1.0](https://github.com/OffchainLabs/nitro/releases/tag/v3.1.0), requires [**`nitro-contracts v2.1.0`**](https://github.com/OffchainLabs/nitro-contracts/releases/tag/v2.1.0) or higher.
```

Comment:
let's also mention this upgrade will also enable Stylus
if they want to opt-out they can skip the wasm module root update and cache manager

Reply:
https://github.com/OffchainLabs/orbit-actions/pull/16/commits/2e7d234fdf1d12ac8b321bb847c9f171a19a9ea4